lambdas/get-addresses/package.json

```diff
@@ -15,7 +15,9 @@
         "@aws-sdk/client-dynamodb": "3.362.0",
         "@aws-sdk/client-ssm": "^3.362.0",
         "@aws-sdk/lib-dynamodb": "3.150.0",
-        "esbuild": "0.14.14"
+        "@smithy/node-http-handler": "2.3.1",
```

Comment:
Is there an approval for this package to be used in the lambda?

Reply:
I tried install this `@aws-sdk/node-http-handler` got some depreciation warning in favor of the one above AWS SDK uses this see https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md